SocialCareCaseViewerApi.Tests/V1/Helpers/CaseStatusHelper.cs

```diff
@@ -26,16 +22,55 @@ public static class CaseStatusHelper
         public static (CaseStatus, SocialCareCaseViewerApi.V1.Infrastructure.Person) SavePersonWithCaseStatusToDatabase(
           DatabaseContext databaseContext)
         {
-            var caseStatusType = TestHelpers.CreateCaseStatusType();
-            var caseStatusSubtype = TestHelpers.CreateCaseStatusSubtype(typeId: caseStatusType.Id);
+            var caseStatusType = TestHelpers.CreateCaseStatusType(id: 10);
+            var caseStatusSubtype = TestHelpers.CreateCaseStatusSubtype(typeId: caseStatusType.Id, id: 20);
+            var person = TestHelpers.CreatePerson(3);
+            var csus = TestHelpers.CreateCaseStatus(personId: 3, typeId: caseStatusType.Id, subtypeId: caseStatusSubtype.Id, startDate: DateTime.Today, notes: "Testing");
+
+
+            databaseContext.CaseStatusTypes.Add(caseStatusType);
+            databaseContext.CaseStatusSubtypes.Add(caseStatusSubtype);
+            databaseContext.Persons.Add(person);
+            databaseContext.CaseStatuses.Add(csus);
+
+            databaseContext.SaveChanges();
+
+            return (csus, person);
+        }
+
+        public static (CaseStatus, SocialCareCaseViewerApi.V1.Infrastructure.Person) SavePersonWithPastCaseStatusToDatabase(
+          DatabaseContext databaseContext)
+        {
+            var caseStatusType = TestHelpers.CreateCaseStatusType(id: 10);
+            var caseStatusSubtype = TestHelpers.CreateCaseStatusSubtype(typeId: caseStatusType.Id, id: 20);
+            var person = TestHelpers.CreatePerson(3);
+            var csus = TestHelpers.CreateCaseStatus(personId: 3, typeId: caseStatusType.Id, subtypeId: caseStatusSubtype.Id, startDate: DateTime.Today.AddDays(-2), endDate: DateTime.Today.AddDays(-1), notes: "Testing");
```

Comment:
Do we need to specify the IDs for all of these? Is this to combat the seeding in `schema.sql` for the subtype?

Reply:
Exactly because of that 😂